composer.json

```diff
@@ -17,7 +17,8 @@
         "symfony/console": "~2.3.10|~2.4.2|~2.5",
         "nikic/php-parser": "~1.0",
         "dnoegel/php-xdg-base-dir": "0.1",
-        "jakub-onderka/php-console-highlighter": "0.3.*"
+        "jakub-onderka/php-console-highlighter": "0.3.*",
+        "jeremeamia/SuperClosure": "@stable"
```

Comment:
agreed. but we're likely not using it at all, so i didn't worry too much about it :)